code/mod_table/mod_table.cpp

```diff
@@ -538,6 +540,14 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$Shockwaves Always Damage Bombs:")) {
+			stuff_boolean(&Shockwaves_always_damage_bombs);
+		}
+
+		if (optional_string("$Shockwaves Damage Non-ships Once:")) {
```

Comment:
I would call this `$Shockwaves Damage Bombs and Asteroids Once:` otherwise modders might wonder if ships are damaged more than once.

It's hard to think of a good table string here.  The bool variable name is more accurate but not very explanatory.

Reply:
How about Shockwaves_damage_all_obj_types_once ?

Reply:
There you go, that sounds good. :+1: